lib/tether/tether.go

```diff
@@ -212,6 +217,15 @@ func (t *tether) Start() error {
 		}
 		extraconfig.Encode(t.sink, t.config)
 
+		log.Info("Populating block device list now.")
+		BlockDevices, err := fs.NewBlockDevices(linuxBlockDevicePath)
+		DeviceMap := BlockDevices.DevicesByLabel()
```

Comment:
You are shadowing DeviceMap here so either make this an assignment or remove the declaration